operator/roles/ansible-service-broker/templates/broker-operator.monitor-role.yaml.j2

```diff
@@ -0,0 +1,17 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  # TODO this should be a clusterrole
```

Comment:
I don't see why this needs to be a clusterrole. Maybe remove the comment if it's not needed?